src/utils/events/index.ts

```diff
@@ -265,7 +266,7 @@ export function parseEvents(
  * @returns DeployContractResponse | UDC Event Response data
  */
 export function parseUDCEvent(txReceipt: InvokeTransactionReceiptResponse) {
-  if (!txReceipt.events) {
+  if (!txReceipt.events?.length) {
```

Comment:
The old implementation will never happen (`events` are mandatory by the TS). Either we can check the `length` as I did in the PR and throw an error if there are no events, or we can remove the error handling part, and it will just return the response with the undefined values (it worked like that in the prev. implementation). In both ways, all tests pass. @penovicp thoughts? 🤔